### PR TITLE
fix a typo in Security.java

### PR DESCRIPTION
**Pull request by dh-cloud**
"path.conf'" has an extra “'”

### CODE REVIEW

**Comment by grcevski** [comment]
@elasticsearchmachine test this please

**Review by grcevski — APPROVED** [review]
LGTM! Thanks for fixing this.

File: docs/changelog/89248.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pr: 89248
+summary: fix "path.conf'" typo in Security.java
+area: Bootstrap
```
Comment (grcevski) [inline docs/changelog/89248.yaml:3]: We should change the area to one of the known areas. I think it should be Engine.

Comment (dh-cloud) [reply docs/changelog/89248.yaml]: Thanks for your opinion，I have modified it.

**Comment by grcevski** [comment]
@elasticsearchmachine test this please

**Comment by grcevski** [comment]
Thanks for your contribution @dh-cloud!

**Pull request merged**